tasties_app/models.py

```diff
@@ -105,6 +106,25 @@ class Comment(models.Model):
     publication_date = models.DateField(default=timezone.now)
     comment_text = models.CharField(max_length=512, validators=[MinLengthValidator(1)])
 
+    def __str__(self):
+        try:
+            a_id = str(self.author_id)
+            comment = self.comment_text
+            date1 = str(self.publication_date)
+            string = a_id+' '+comment+' '+date1
+            return string
+        except ValidationError as e:
+            raise (e)
+
+    # function to add comment with length check
+    def add_comment(self, text):
+        if len(text) < 1:
+            raise ("len the text is illegal")
```

Comment:
Please fix the error message to something like this:
"Illegal text length" or "Empty comment"

Reply:
When prompting an error or a warning, you want to be as clear as you can, so the user won't get frustrated.
The error message should be explicit and make sense for the user.
Please change it accordingly